src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -55,5 +58,12 @@ public PersonCard(Person person, int displayedIndex) {
         person.getTags().stream()
                 .sorted(Comparator.comparing(tag -> tag.tagName))
                 .forEach(tag -> tags.getChildren().add(new Label(tag.tagName)));
+        String remarkValue = person.getRemark().value;
+        if (remarkValue != null && !remarkValue.trim().isEmpty()) {
+            remark.setText(remarkValue);
+            remark.setManaged(true);
```

Comment:
What does the `setManaged` do here?

Reply:
it renders/unrenders the remark label entirely so that it is hidden if it is empty